process/process_linux.go

```diff
@@ -809,6 +810,11 @@ func AllProcesses(cpuWait time.Duration) ([]*FilledProcess, error) {
 		return nil, fmt.Errorf("could not collect pids: %s", err)
 	}
 
+	containers, err := docker.GetContainerStat()
+	if err != nil {
+		containers = make(map[string]ContainerStat)
```

Comment:
It seems right to not fail everything if this call falls but we should at least log the error